graylog2-web-interface/src/components/sidecars/sidecars/SidecarStatus.jsx

```diff
@@ -2,19 +2,24 @@ import React from 'react';
 import createReactClass from 'create-react-class';
 import PropTypes from 'prop-types';
 import lodash from 'lodash';
-import { Col, Row } from 'react-bootstrap';
+import { Button, Col, Row } from 'react-bootstrap';
 
 import SidecarStatusEnum from 'logic/sidecar/SidecarStatusEnum';
 import commonStyles from 'components/sidecars/common/CommonSidecarStyles.css';
 
 import SidecarStatusFileList from './SidecarStatusFileList';
+import VerboseMessageModal from './VerboseMessageModal';
 
 const SidecarStatus = createReactClass({
   propTypes: {
     sidecar: PropTypes.object.isRequired,
     collectors: PropTypes.array.isRequired,
   },
 
+  getInitialState() {
+    return { collectorName: "", collectorVerbose: "" };
```

Comment:
ESLint complains about this line, as we use `single` quotes for strings (see https://eslint.org/docs/rules/quotes#single). You can fix it with:

```suggestion
    return { collectorName: '', collectorVerbose: '' };
```